src/components/WeightageList.jsx

```diff
@@ -1,41 +1,18 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 
-import { useParams } from "react-router-dom";
-import { Select, notification } from "antd";
-import axios from "axios";
+import { Select } from "antd";
 
-import { API_END_POINT } from "../../config";
-
-import { useAuth } from "../context/AuthContext";
 
 const WeightageList = ({
-  taskId,
   taskWeightages,
   handleSaveWeightage,
   handleAddWeightage,
   handleWeightageChange,
   handleDeleteWeightage,
   weightages,
-  setWeighatages
 }) => {
-  const { id: batchId } = useParams();
-  const { token } = useAuth();
  
 
-  const headers = {
-    Authorization: `Bearer ${token.access}`,
-    "Content-type": "application/json",
-  };
-  useEffect(() => {
-    axios
-      .get(`${API_END_POINT}/api/task/${batchId}/list/weightage`, { headers })
```

Comment:
Why am i reviewing these changes again? when this was already merged to dev?

Reply:
The weightage component inside the weightage list is fetched but the usestate assessment view is there and when the student mark section is opened the weightage list is not fetched and it comes as an empty array so I have changed the useeffect assessment view where the weightage list component is there.